lib/gcs_signer.ex

```diff
@@ -5,12 +5,14 @@ defmodule GcsSigner do
 
   @base_url "https://storage.googleapis.com"
 
+  @otp_greater_21? :erlang.system_info(:otp_release) >= '21'
+
   @type sign_url_opts :: [
-    verb: String.t,
-    md5_digest: String.t,
-    content_type: String.t,
-    expires: integer
-  ]
+          verb: String.t(),
```

Comment:
May I ask if there is a reason why indentation and method call format has changed here? I would prefer keeping the original indentation to match with other parts of our codebase.